policies/release-requirements.md

```diff
@@ -0,0 +1,92 @@
+- There is no remaining refactoring required by OMC or OTC for the release.
+- There are no remaining API changes required for the release.
+  _This applies only to beta releases of a major release as API changes
+  are not allowed on minor releases._
+- There are no remaining API additions required for the release.
+- All untriaged issues older than 2 days must be triaged.
+- All regressions on the development branch from which the
+  release is to be done must have a milestone assigned by OTC.
+  _Regressions are considered from the previous stable release. The process
+  how OTC assigns the milestone is not exactly defined on purpose._
+- All issues or pull requests with beta milestone for the beta release
+  are closed.
+- There are no outstanding untriaged Coverity issues.
+- Coveralls coverage has not decreased overall from the previous release.
+- The CI must pass on the tip of the development branch before the release
+  commits are added to the tree, including the daily CI builds.
+- The tree must be frozen at least for one day, which also ensures the daily CI
+  builds run on the development tree tip.
+- In case of the first beta release the OTC should explicitly approve
+  that the source is ready for a release with a vote.
+
+Stable Releases
+---------------
+
+For stable releases the requirements are basically the same as the release
+requirements for the beta releases but they are repeated here for clarity.
+As the release comes after the beta releases there is no need to repeat the
+stability requirements as those should be held already by the beta releases.
+
+- All untriaged issues older than 2 days must be triaged.
+- All regressions on the development branch from which the
+  release is to be done must have a milestone assigned by OTC.
+  _Regressions are considered from the previous stable release. The process
+  how OTC assigns the milestone is not exactly defined on purpose._
+- All issues or pull requests with milestone for the release are closed.
+- There are no outstanding untriaged Coverity issues.
+- Coveralls coverage has not decreased overall from the previous release.
+- The CI must pass on the tip of the development branch before the release
+  commits are added to the tree, including the daily CI builds.
+- The tree must be frozen at least for one day, which also ensures the daily CI
+  builds run on the development tree tip.
+- In case of the first stable release from the development branch the OTC should
+  explicitly approve that the source is ready for a release with a vote.
+
+Triage Process
+--------------
+
+The issues and pull requests in GitHub must be assigned a `triaged: *` label by
+an OTC member according to what is the type (feature, bug, documentation,
+refactoring, ...) of the issue or pull request. When the triage happens for an
+issue or pull request that is a bug/bug fix, it must be assessed whether the
+bug is a regression or not. In general regressions are not allowed in stable
```

Comment:
We don't do this assessment at the moment - and often it will not be obvious whether it is or not. I don't think we have a way of identifying regressions?